integration_tests/ctesque/src/sharedTest/java/android/graphics/ColorTest.java

```diff
@@ -0,0 +1,25 @@
+package android.graphics;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.annotation.internal.DoNotInstrument;
+
+/** Compatibility tests for {@link Color} */
+@DoNotInstrument
+@RunWith(AndroidJUnit4.class)
+public class ColorTest {
+  @Test
+  public void HSVToColorValueShouldBePinned() {
+    assertThat(Color.HSVToColor(new float[] {0f, 0f, -1.0f})).isEqualTo(Color.BLACK);
```

Comment:
Maybe removing tests from shadows directory? This test file will be executed both in Robolectric environment and Emulator.

Reply:
Removed, thanks!